src/ProjectTemplates/scripts/.gitignore

```diff
@@ -1,17 +1,2 @@
 package-lock.json
-tmp/
-CustomHive/
-angular/
-blazorserver/
-blazorwasm/
-mvc/
-mvcorgauth/
-razor/
-react/
-reactredux/
-web/
-webapp/
-webapi/
-webapimin/
-worker/
-grpc/
\ No newline at end of file
+**/
```

Comment:
Do we really want to ignore the `*.ps1` files in the scripts/ folder❔

Reply:
The intention was this would ignore sub-directories.

Reply:
Verified this isn't ignoring files, just sub-dirs.